src/src/rx-serial/SerialHoTT_TLM.cpp

```diff
@@ -12,7 +13,7 @@
 #define HOTT_CMD_DELAY 1        // 1 ms delay between CMD byte 1 and 2
 #define HOTT_WAIT_TX_COMPLETE 2 // 2 ms wait for CMD bytes transmission complete
 
-#define DISCOVERY_TIMEOUT 40000 // 40s device discovery time
+#define DISCOVERY_TIMEOUT 30000 // 30s device discovery time
```

Comment:
Was it intentional that the previous change got reverted in this commit? The reasoning in the PR text sounds like that is still required (or I misread it).